src/app/Scenes/Home/Components/useImpressionsTracking.ts

```diff
@@ -0,0 +1,68 @@
+import { ContextModule } from "@artsy/cohesion"
+import HomeAnalytics from "app/Scenes/Home/homeAnalytics"
+import { useFeatureFlag } from "app/store/GlobalStore"
+import { useEffect, useRef, useState } from "react"
+import { ViewToken, ViewabilityConfig } from "react-native"
+import { useTracking } from "react-tracking"
+
+type TrackableItem = { id: string; index: number }
+export const useItemsImpressionsTracking = ({
```

Comment:
nitpick: To make it easier to read:

```suggestion
type TrackableItem = { id: string; index: number }

export const useItemsImpressionsTracking = ({
```